backend/src/main/java/com/benine/backend/http/CameraRequestHandler.java

```diff
@@ -0,0 +1,42 @@
+package com.benine.backend.http;
+
+import org.eclipse.jetty.server.Request;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+/**
+ * Handles all requests requiring the camera ID.
+ */
+public abstract class CameraRequestHandler extends RequestHandler {
+
+  /**
+   * Fetches camera id from http exchange.
+   * @param request the request to fix the id from.
+   * @return the id of the camera.
+   */
+  public int getCameraId(Request request) {
+    // Get path
+    Pattern pattern = Pattern.compile(".*/(\\d*)/.*");
+    String path = request.getPathInfo();
+
+    Matcher m = pattern.matcher(path);
+
+    if (m.matches()) {
```

Comment:
You can clean this up by doing something like "return m.matches ? Intteger.parseInt(m.group(1) : -1"